kratos/utilities/cpp_tests_utilities.cpp

```diff
@@ -69,8 +71,48 @@ void Create2DGeometry(
 /***********************************************************************************/
 /***********************************************************************************/
 
+void CreatePure2DGeometry(ModelPart& rModelPart)
```

Comment:
If you aim to add these to the common test utilities, I'd try to find a more self-descriptive name.

Reply:
Suggestion?, this was the best I could think.